.github/scripts/check_newsfragments.py

```diff
@@ -1,57 +1,33 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+"""
+PRs must contain a newsfragment that reference an opened issue
+"""
 
-# PRs must contain a newsfragment that reference an opened issue
 import re
-import sys
 import json
+import argparse
 from pathlib import Path
 from subprocess import run
 from urllib.request import urlopen, Request, HTTPError
 from concurrent.futures import ThreadPoolExecutor
-import logging
 
-logging.basicConfig(level=logging.INFO)
-ROOT_LOGGER = logging.getLogger()
 
 # If file never existed in master, consider as a new newsfragment
 # Cannot just git diff against master branch here given newsfragments
 # removed in master will be considered as new items in our branch
 # --exit-code makes the command exit with 0 if there are changes
 BASE_CMD = "git log origin/master --exit-code --".split()
 
-
-def parse_args():
-    import argparse
-
-    parser = argparse.ArgumentParser("check_news_fragment")
-    parser.add_argument("branch_name", help="current branch name", type=str)
-    return parser.parse_args()
-
-
-args = parse_args()
-BRANCH_NAME = args.branch_name
-
-# Regex pattern to match for release branch.
-# Release branch don't have to create a news fragment
-RELEASE_BRANCH_NAME_PATTERN = r"[0-9]+.[0-9]+"
-
-ROOT_LOGGER.info(f"current branch {BRANCH_NAME}")
-if re.match(RELEASE_BRANCH_NAME_PATTERN, BRANCH_NAME) is not None:
-    print("Release branch detected, ignoring newsfragment checks")
-    sys.exit(0)
+# Ignore master given we compare against it !
+# Also ignore release branch (e.g. `2.11`) that don't have to create newsfragments
+IGNORED_BRANCHES_PATTERN = r"(master|[0-9]+.[0-9]+)"
```

Comment:
```suggestion
IGNORED_BRANCHES_PATTERN = r"(master|[0-9]+\.[0-9]+)"
```
the release branch pattern is `2.9` for example (the `.` is important)